### PR TITLE
feat: :rocket: Created page with HTML and CSS

### PR DESCRIPTION
**Pull request by EoinMcD**
# Description
Added a new page using HTML and styled with CSS
The page includes: 

- Text
- List
- Image
- Video
- Link
- Table
- Form

#Testing
Navigate to /about 

# Screenshots
Screenshot 2021-04-19 at 11 38 07
Screenshot 2021-04-19 at 11 38 18

### CODE REVIEW

File: app/assets/stylesheets/about.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+.name 
+h1 {
+    font-size: 150px ;
+    text-align:center;
+    font-family:fantasy;
+    padding-top:50px;
+    position: relative;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 150px 0px 150px;
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: You can use shorthand to more quickly set these duplicated values.

For the two above you could shorten these to

```
margin: 0;
padding: 0 150px;

You can read more about CSS shorthand here: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties
```

File: app/assets/stylesheets/about.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    display:inline-flex;
+    justify-content: center;
+    background-color:#55a8c9; 
+    flex-wrap:nowrap; 
+    width:100%
+    
+}
+
+.name img{
+    padding-right:0px;
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: You don't need to specify px when setting to 0, as it's 0 it doesn't matter what measure you're using.

File: app/assets/stylesheets/about.scss [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+.divider
+p {
+    font-size: 50px;
+    text-align:center;
+    padding-top:0px;
+    background-color:#3AAFA9;
+    color:white;
+    margin: 0;
+    padding: 0;
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: Here you have specified padding-top twice, as it is included in padding.

File: app/assets/stylesheets/about.scss [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.media {
+    display: inline-flex;
+    justify-content: space-around;
+    flex-direction: row;
+    width:100%;
+    background-color:bisque;
+    margin: 0;
+    padding: 0;
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: Out of curiosity, I see you're specifying margin and padding to be 0 in a lot of places. Is this necessary? If so, could there be a different way to achieve the same thing, such as setting margin and padding to 0 at a higher level?

File: app/assets/stylesheets/about.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    font-size: 16px;
+     font-weight: 100;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+}
+
+.body tr{
+    background: #f4f7f8;
+    border-bottom: 1px solid #FFF;
+    margin-bottom: px;
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: No value set here :)

File: app/assets/stylesheets/about.scss [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+.signup {
+    display:flex;
+    justify-content: center;
+    background-color:bisque;
+}
+
+.signup p{
+    font-size: 30px;
+}
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: As this is a SCSS file, you can nest these, I believe there are other examples within this file. 

Eg. 

```
.signup {
    display: flex;
    justify-content: center;
    background-color: bisque;

   p {
    font-size: 30px;
   }
}
```

File: app/views/about/Index.html.erb [hunk 7]
```diff
@@ -0,0 +1,73 @@
+<link href ='app/assets/stylesheets/aboutme.scss' type='text/css' rel='stylesheet'>
+<div class="name">     
+    <img width='300px' height='300px' src= 'https://cdn.filestackcontent.com/AchUBPpbtR12UdA8r3ilwz/security=policy:eyJleHBpcnkiOjIyNDE3MzU4MTksImNhbGwiOlsicmVhZCIsImNvbnZlcnQiXSwiaGFuZGxlIjoiWHg5N2dVT1BSSXFjanJvWkZJcHgifQ==,signature:87dd7e34f59faed2f03dc50d44bfff58f2bce4dfc2de26718cadd7fdc7610e1a/cache=expiry:max/resize=w:600,h:600,fit:crop,align:faces/rotate=d:exif/Xx97gUOPRIqcjroZFIpx '/>
```
Comment (meganduffy) [inline app/views/about/Index.html.erb]: You're using inline styling here to set the width and height of this image. While this is not wrong, it's good to note that these values could also live within the CSS file.

File: app/views/about/Index.html.erb [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        <ul>
+            <li>HTML?</li>
+            <li>CSS?</li>
+            <li>Java</li>
+            <li>Ruby</li>
+            <li>C++</li>
+            <li>Gamemaker Language</li>
+        </ul>
+        <table class = "table">
+            <h2>Projects I have worken on</h2>
```
Comment (meganduffy) [inline app/views/about/Index.html.erb]: I think you have a typo there ;)

File: app/views/about/Index.html.erb [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+            <input type="email"name="Email">
+            <input type="submit"name="Name">
+        </form>
+    </div>
+    <div class='footer'>
+        <a href='https://www.linkedin.com/in/eoin-mcdonough-a2a3921b8/' target="_blank">LinkedIn</a>
+        <a href='https://wpengine.pingboard.com/users/7597031' target="_blank">Pingboard</a>
+        <a href="https://github.com/EoinMcD" target='_blank'> Github</a>
+   
+</div>
```
Comment (meganduffy) [inline app/views/about/Index.html.erb]: No newline at EOF

**Review by meganduffy — COMMENTED** [review]
Great work Eoin, the concept is great! There are a few comments to address but overall it's looking good. 

Two observations I would mention:

- The ticket calls out a video as one of the ACs, I see you've used a gif which isn't far off at all but it also isn't what is called out in the ticket. 
- There are quite a few formatting issues in this PR (spaces where they shouldn't be, or too many, inconsistent newlines) I haven't pointed them out in the comments but you will want to either manually tidy them up or run your code through a linter. I use the Prettier formatter extension in VSCode, I believe you also have the same one installed.

File: app/assets/stylesheets/about.scss (hunk 1 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: I can't believe I didnt spot this, fixed now, thanks for the info too

File: app/assets/stylesheets/about.scss (hunk 2 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: Fixed

File: app/assets/stylesheets/about.scss (hunk 3 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: Didn't see this, fixed now

File: app/assets/stylesheets/about.scss (hunk 5 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: My bad haha, fixed

File: app/assets/stylesheets/about.scss (hunk 6 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: Ahh, when you said nesting before I didn't realise this is what you meant. I went through the file and nested all I could, looks much better, thanks!

File: app/views/about/Index.html.erb (hunk 7 above)
Comment (EoinMcD) [reply app/views/about/Index.html.erb]: Don't know how that happened, fixed now

File: app/views/about/Index.html.erb (hunk 8 above)
Comment (EoinMcD) [reply app/views/about/Index.html.erb]: Yup, fixed :)

File: app/views/about/Index.html.erb (hunk 9 above)
Comment (EoinMcD) [reply app/views/about/Index.html.erb]: Added newline

File: app/assets/stylesheets/about.scss (hunk 4 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: Good idea, I made a new class and added margin and padding to it, and applied it at a higher level

File: app/assets/stylesheets/about.scss [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+  img {
+    padding-right: 0;
+    margin-right: 0;
+    padding-top: 75px;
+    width: 300px;
+    height: 300px;
+  }
+}
+
+.padding {
```
Comment (meganduffy) [inline app/assets/stylesheets/about.scss]: The classname padding here is a bit misleading as it's setting the padding and the margin to 0 - how about something like `.thin` or `.narrow`?

**Review by meganduffy — COMMENTED** [review]
Not a blocker but something to keep in mind when creating classnames!

File: app/assets/stylesheets/about.scss (hunk 10 above)
Comment (EoinMcD) [reply app/assets/stylesheets/about.scss]: Thats a good point, didn't think of that, I'll change it to .thin

**Review by meganduffy — APPROVED** [review]

**Pull request merged**